### PR TITLE
Initial Sync of SEP definitions from redcap

### PR DESCRIPTION
**Pull request by yatharthranjan**
- The project name is SEP_TEST. This should be the same in MP too (when created).
- Similar to aRMT definitions but adds a new field type called `eval`. These fields should contain the branching logic for evaluating the rest of the form. These should not be displayed in the UI.

Closes: https://github.com/RADAR-base/hydra-kratos/issues/46

### CODE REVIEW

File: projects/SEP_TEST/consent/SEP_TEST_consent_v1.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+[
+    {
+        "field_name":"sep_consent_field_1",
+        "form_name":"sep_consent",
+        "section_header":"",
+        "field_type":"info",
```
Comment (this-Aditya) [inline projects/SEP_TEST/consent/SEP_TEST_consent_v1.json:6]: Should the type be `"field_type": "descriptive"`, as `"field_type": "info"` is not present in the REDCap data dictionary.

Then we could set the `select_choices_or_calculations.label`  to "field_label" and `select_choices_or_calculations.code` as "section_header", which results in two fields.

File: projects/SEP_TEST/consent/SEP_TEST_consent_v1.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        "select_choices_or_calculations":[
+            {
+                "code":"0",
+                "label":"No"
+            },
+            {
+                "code":"1",
+                "label":"Yes"
+            }
+        ],
```
Comment (this-Aditya) [inline projects/SEP_TEST/consent/SEP_TEST_consent_v1.json:46]: Should this be in the format:  `"1, Answer 1 | 2, Answer 2 |
3, Answer 3 | … N, Answer N."`?

**Review by this-Aditya — COMMENTED** [review]
Hi @yatharthranjan,

The PR looks great overall. I’ve added some comments with the assumption that we’re following the REDCap data dictionary format. 
If that’s not the case and we’re customizing it slightly, feel free to ignore these comments.

Thanks!

File: projects/SEP_TEST/eligibility/SEP_TEST_eligibility_v1.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "field_name":"sep_eligibility_field_1",
+        "form_name":"sep_eligibility",
+        "section_header":"",
+        "field_type":"text",
+        "field_label":"How old are you?",
+        "select_choices_or_calculations":"",
+        "field_note":"",
+        "text_validation_type_or_show_slider_number":"number",
```
Comment (this-Aditya) [inline projects/SEP_TEST/eligibility/SEP_TEST_eligibility_v1.json]: Should we use `integer` here, as the 'integer' validation only allows whole numbers, while the `number` validation allows numbers with decimals?

File: projects/SEP_TEST/study_info/SEP_TEST_study_info_v1.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
+[
+    {
+        "field_name":"study_info_field_1",
+        "form_name":"sep_study_info",
+        "section_header":"",
+        "field_type":"info",
```
Comment (this-Aditya) [inline projects/SEP_TEST/study_info/SEP_TEST_study_info_v1.json:6]: Similarly, could this be set to `descriptive`?

File: projects/SEP_TEST/consent/SEP_TEST_consent_v1.json (hunk 1 above)
Comment (yatharthranjan) [reply projects/SEP_TEST/consent/SEP_TEST_consent_v1.json:6]: The apps in RADAR support additional types than redcap. You can see some types supported here for the Questionnaire app - https://github.com/RADAR-base/RADAR-Questionnaire/tree/master?tab=readme-ov-file#questionnaire-input-types

`info` type is for simple text info to be shown in the app. The conversion from redcap to RADAR type is done using scripts (one of which is being added [here](https://github.com/RADAR-base/radar-self-enrolment-definitions/pull/3)) based on the `field_type` and `field_annotation` in redcap metadata.

File: projects/SEP_TEST/consent/SEP_TEST_consent_v1.json (hunk 2 above)
Comment (yatharthranjan) [reply projects/SEP_TEST/consent/SEP_TEST_consent_v1.json:46]: Yes, redcap codes them as you mentioned but for easy parsing the app in RADAR use code/label as above, modified by the script. You can look at the previous definitions in radar for questionnaires to get an idea - https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/tree/master/questionnaires

File: projects/SEP_TEST/eligibility/SEP_TEST_eligibility_v1.json (hunk 3 above)
Comment (yatharthranjan) [reply projects/SEP_TEST/eligibility/SEP_TEST_eligibility_v1.json]: Yes good point.

File: projects/SEP_TEST/study_info/SEP_TEST_study_info_v1.json (hunk 4 above)
Comment (yatharthranjan) [reply projects/SEP_TEST/study_info/SEP_TEST_study_info_v1.json:6]: Yes could be, but unless there is html involved, we tend to use simple `info` type.

File: projects/SEP_TEST/consent/SEP_TEST_consent_v1.json (hunk 1 above)
Comment (this-Aditya) [reply projects/SEP_TEST/consent/SEP_TEST_consent_v1.json:6]: Okay, fine. Thanks for the references.

**Review by this-Aditya — APPROVED** [review]
Thanks, LGTM.

**Review by afolarin — APPROVED** [review]
LGTM

**Pull request merged**